packages/mattermost-redux/src/actions/threads.ts

```diff
@@ -31,18 +31,36 @@ import {forceLogoutIfNecessary} from './helpers';
 
 type ExtendedPost = Post & { system_post_ids?: string[] };
 
-export function getThreads(userId: string, teamId: string, {before = '', after = '', perPage = ThreadConstants.THREADS_CHUNK_SIZE, unread = false, totalsOnly = false} = {}) {
+function fetchThreads(userId: string, teamId: string, {before = '', after = '', perPage = ThreadConstants.THREADS_CHUNK_SIZE, unread = false, totalsOnly = false, threadsOnly = false} = {}) {
     return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
-        let userThreadList: undefined | UserThreadList;
+        let data: undefined | UserThreadList;
 
         try {
-            userThreadList = await Client4.getUserThreads(userId, teamId, {before, after, perPage, extended: false, unread, totalsOnly});
+            data = await Client4.getUserThreads(userId, teamId, {before, after, perPage, extended: false, unread, totalsOnly, threadsOnly});
         } catch (error) {
             forceLogoutIfNecessary(error, dispatch, getState);
             dispatch(logError(error));
             return {error};
         }
 
+        return {data};
+    };
+}
+
+export function getThreads(userId: string, teamId: string, {before = '', after = '', perPage = ThreadConstants.THREADS_CHUNK_SIZE, unread = false} = {}) {
```

Comment:
I'm gonna steal this way of defining objects in the signature of the function :laughing:

Reply:
destructuring an object in function args is a neat trick :)